components/runtime/smalltalk/squeak/Makefile

```diff
@@ -71,7 +71,7 @@ include ../../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		squeak
 COMPONENT_VERSION=	4.20.10
-COMPONENT_REVISION=	3
+COMPONENT_REVISION=	4
 COMPONENT_SUMMARY=	The Squeak Virtual Machine
```

Comment:
You don't need to increment COMPONENT_REVISION because the last PR hasn't been published.